app/controllers/hackdays_controller.rb

```diff
@@ -0,0 +1,48 @@
+class HackdaysController < ApplicationController
+	
+	def create
+		@hackday = Hackday.new(hackday_params)
+		if @hackday.save
+			flash[:success] = "successfully added a hackday"
+			redirect_to @hackday
+		else
+			flash[:danger] = "failed to add a hackday"
+			redirect_to root_url
+		end
+
+	end
+
+	def index
+		@new_hackday = Hackday.new
+		@hackdays = Hackday.all
+		@current_hackday = @hackdays.last #most recent one	
+		@past_hackdays = @hackdays.first (@hackdays.size - 1)
```

Comment:
there are quite a few nicer solutions to this.

solution one:

@current_hackday= @hackdays.delete(@hackdays.last)
@past_hackays = @hackdays

solution 2:

@past_hackdays = @hackdays[0..-1]